ZeldaFullEditor/Gui/ExtraForms/AsmPlugin.cs

```diff
@@ -513,7 +515,7 @@ private void SavePatches()
             StringBuilder generatedAsmFile = new StringBuilder();
             generatedAsmFile.AppendLine("lorom");
 
-            generatedAsmFile.AppendLine("org $308000"); // need to do something about it
+            generatedAsmFile.AppendLine("org $3C8000"); // need to do something about it
```

Comment:
I think we should either make this a variable the user can change (the expanded bank for the file) or a constant with the other expanded regions.